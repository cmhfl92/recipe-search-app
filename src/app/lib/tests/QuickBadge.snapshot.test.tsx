import QuickBadge from '@/app/components/quickBadge';
import { render } from '@testing-library/react';

describe('QuickBadge Snapshot', () => {
  it('renders wuick badge corerctly for <= 15 minutes', () => {
    const { asFragment } = render(<QuickBadge time={10} />);
    expect(asFragment()).toMatchSnapshot();
  });

  it('renders nothing for time > 15 minutes', () => {
    const { asFragment } = render(<QuickBadge time={20} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
