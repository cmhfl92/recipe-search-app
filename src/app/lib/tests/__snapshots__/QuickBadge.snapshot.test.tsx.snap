// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuickBadge Snapshot renders nothing for time > 15 minutes 1`] = `<DocumentFragment />`;

exports[`QuickBadge Snapshot renders wuick badge corerctly for <= 15 minutes 1`] = `
<DocumentFragment>
  <span
    class="text-green-600"
    title="Quick Recipe"
  >
    ğŸš€
  </span>
</DocumentFragment>
`;
